purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -330,23 +332,28 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
             receiptId,
             amazonUserID,
             { normalizedProductID ->
+                val receiptInfo = ReceiptInfo(
+                        productID = normalizedProductID,
+                        price = price?.takeUnless { it == 0.0 },
```

Comment:
why do we have to worry about the price = 0 case?

Reply:
unity would give price of 0 if they can't figure out the price of a product. It's better to not send any price to the backend than sending 0